lib/features/extend_task_graph.js

```diff
@@ -28,8 +27,14 @@ ExtendTaskGraph.prototype = {
     var scheduler = taskHandler.runtime.scheduler;
 
     // Raw tar stream for the content.
+    var contentStream;
     try {
-      var contentStream = yield container.copy({ Resource: graphPath });
+      contentStream = await (new Promise((accept, reject) => {
+        return container.copy({ Resource: graphPath }, function(err, data) {
```

Comment:
mistake in looking at some dockerode code that was callback based and forgetting we are wrapping this in a promise proxy elsewhere.  Will fix it up.